letsencrypt/data/run.sh

```diff
@@ -124,6 +124,6 @@ else
 fi
 
 # Get the last modified cert directory and copy the cert and private key to store
-CERT_DIR=$(ls -td $CERT_DIR/live/* | head -1)
+CERT_DIR=$(realpath(ls -td $CERT_DIR/live/*/ | head -1))
```

Comment:
(outside of the typo here) README file was being returned, assume it gets laid down after the cert files. Using trailing slash to only look for directories, and `realpath` to remove the trailing slash.

Code check fails because of `ls` call (really.. it's too damn much to try to sort directories by modification date using the `find` command) and not-quoted parameter (we can fix that if needed, no big deal).

`CERT_DIR=$(realpath "$(ls -td $CERT_DIR/live/*/ | head -1)")` would be the quoted variant.

Reply:
> CERT_DIR=$(realpath "$(ls -td $CERT_DIR/live/*/ | head -1)") would be the quoted variant.

let do this, so we can merge if he pass the CI.